drivers/wireless/ieee80211/bcm43xxx/bcmf_netdev.c

```diff
@@ -663,6 +698,17 @@ static int bcmf_ifup(FAR struct net_driver_s *dev)
   /* Enable the hardware interrupt */
 
   priv->bc_bifup = true;
+
```

Comment:
let's goto errout_in_critical_section and remove the ret check at line 703

Reply:
Done